spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -4,6 +4,104 @@ This section goes into more detail about how you can work with Spring Cloud Stre
 It covers topics such as creating and running stream applications.
 --
 
+== What's New in 2.0?
+Spring Cloud Stream introduces quite a number of new features, enhancements and changes. The following sections outline most notable once. 
+
+=== New Features and Components
+
+==== Polling Consumer
+Introduction of _polled consumers_, where the application can control message processing rates. Please refer to the appropriate section for more details.
+You can also read this blog for more details https://spring.io/blog/2018/02/27/spring-cloud-stream-2-0-polled-consumers
+
+==== Micrometer support
+
+Metrics has been switched to use https://micrometer.io/. `MeterRegistry` is also provided as a bean so custom application can autowire it to capture custom metrics.
+Please refer to the appropriate section for more details
```

Comment:
Asciidoc `https://micrometer.io/[Micrometer].`

>refer to the appropriate section for more details
  
 We usually add a link so they can easily jump.